pkg/verify/testdata/manifests/only-first-used.yaml

```diff
@@ -0,0 +1,112 @@
+# NOTE: This file is parsed by CVO's lib/Manifest ParseManifests method which
```

Comment:
The fact that ParseManifests will load two manifests with identical kind/namespace/name without complaining is probably a bug.  Can we pick different names for these two ConfigMaps, and file a follow-up ticket about failing noisly on that sort of conflict?

Reply:
I changed the manifests names to match the original test data - which was my original intent - which do differ.

I post a PR to fix ParseManifests shortly.